_data/providers.json

```diff
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Get Geyser as a plugin, as in the setup guide. Use the 19131 port and the server ip and connect with the same IP and port as you would on Java. You can also ask for a new port."
```

Comment:
Use port 19131, but then connect with the same port as on Java?
Sorry, that doesn't make sense. Additionally, please use the same layout as other entries.

Reply:
I fixed it